### ISSUE TITLE
Feature: Create app's navbar

### ISSUE THREAD

**Issue by StraykerPL** [StraykerPL/Binder#19]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

![image](https://user-images.githubusercontent.com/45916090/218220349-2a5498ce-c9b6-40c1-b381-a904de2c23e8.png)

![image](https://user-images.githubusercontent.com/45916090/218220461-76c56553-01df-48ed-8601-abce77d7076d.png)

**Describe alternatives you've considered**

**Additional context**
Web Frontend
"Show/Hide Columns" modal is OUT OF SCOPE for this ticket.

**Comment by StraykerPL** [comment]
Blocked by #11 #13

**Comment by StraykerPL** [comment]
Unblocked
